client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -306,12 +310,18 @@ const Checkout = React.createClass( {
 	}
 } );
 
+function mapStateToProps ( state ) {
+	const cards = getStoredCards( state );
+	const selectedSite = getSelectedSite( state );
```

Comment:
It looks like we primarily use the selected site id and slug. So let's use selectors for that data. It makes it easier to refactor if the structure of site changes in the future.
```javascript
import { getSelectedSiteId } from 'state/ui/selectors';
import { getSiteSlug } from 'state/sites/selectors';
//...
const selectedSiteId = getSelectedSiteId( state );
const selectedSiteSlug = getSiteSlug( state, selectedSiteId );
return {
    cards: getStoredCards( state ),
    selectedSiteId,
    selectedSiteSlug
};
```